src/components/Coins.css

```diff
@@ -0,0 +1,91 @@
+.container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.coins-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(7rem, 1fr));
+}
+
+.coin-card {
+  display: flex;
+  padding: 2rem;
+  flex-direction: column;
+  align-items: center;
+  color: white;
+}
+
+.coins-container > a {
+  background-color: #3e64a7;
+}
+
+.coins-container > a:nth-child(odd) {
+  background-color: #4369b2;
+}
+
+.coin-card > img {
+  width: 3rem;
+}
+
+.coin-card > p {
+  word-wrap: break-word;
+  max-width: 7rem;
+}
+
+.coin-container {
+  display: flex;
+  padding: 2rem;
+  flex-direction: column;
+  align-items: center;
+  background-color: #5687e3;
+  height: 100vh;
+  color: white;
+  overflow: hidden;
+}
+
+.coin-container > img {
+  width: 50%;
+  max-width: 110px;
+}
+
+.coin-container table {
+  table-layout: fixed;
+  width: 100%;
+}
+
+.coin-container table th {
```

Comment:
- [ ] Not bad on using `table`, but please make sure you follow the design, see? in the original design same as you did on the home page, the n+1 item has a different background color, e.g: if the card is white, the next is gray and the following is white, etc..., please  do the same for the detail page has I shown below
<img width="419" alt="1" src="https://github.com/mrcbq/coinmarket/assets/77843050/7dd6ea9b-8a7f-4e78-b6f5-f3e50467d8ef">

Reply:
It's Done.